jupyter-scipy/rockcraft.yaml

```diff
@@ -139,22 +170,75 @@ parts:
       # Clean up
       "${CONDA_BIN}/conda" update -y -q --all
       "${CONDA_BIN}/conda" clean -a -f -y
-
+      
       # Install the jupyter python requirements
+      echo "jupyterlab ==${JUPYTERLAB_VERSION}" >> ${CONDA_DIR}/conda-meta/pinned
+      echo "notebook ==${JUPYTER_VERSION}" >> ${CONDA_DIR}/conda-meta/pinned
+      "${CONDA_BIN}/conda" install -y -q \
+        jupyterlab==${JUPYTERLAB_VERSION} \
+        notebook==${JUPYTER_VERSION}
+      
       cp ${CRAFT_PART_SRC}/components/example-notebook-servers/jupyter/requirements.txt jupyter-requirements.txt
       "${CONDA_BIN}/pip" install --no-cache-dir -r jupyter-requirements.txt
```

Comment:
for consistency with upstream, they are copying requirements in `/tmp/requirements.txt` rather than `jupyter-requirements.txt`